lib/rules/selector-not-notation/README.md

```diff
@@ -0,0 +1,57 @@
+# selector-not-notation
+
```

Comment:
Let's add a description, example and extended description

https://stylelint.io/user-guide/rules/about#readmes

Reply:
I'm going offline for the weekend now, but I'll try to rustle up an extended description early next week unless someone beats me to it.

Reply:
Let's try (taken from [this spec](https://www.w3.org/TR/selectors-4/#negation)):

---

# selector-not-notation

Specify simple or complex notation for `:not()` pseudo-classes.

```css
    a:not(.foo, .bar) {}
/**  ↑
 * This notation */
```

In Selectors Level 3, only a single _simple selector_ was allowed as the argument to `:not()`, whereas Selectors Level 4 allows a _selector list_.

Use:

- `"complex"` to author modern Selectors Level 4 CSS
-  `"simple"` for backwards compatibility with older browsers

The [`fix` option](../../../docs/user-guide/usage/options.md#fix) can automatically fix most of the problems reported by this rule.

---

Reply:
> the argument _to_

of?

Reply:
> the argument to

@jeddy3 of?

Reply:
Let's keep it as it is as it's a copy-paste job from the spec.